python/cugraph-dgl/tests/test_utils.py

```diff
@@ -140,3 +143,36 @@ def test_create_homogeneous_sampled_graphs_from_dataframe():
         de, dd = d_block.edges()
         assert torch.equal(ce, de)
         assert torch.equal(cd, dd)
+
+
+def test_get_source_destination_range():
+    df = get_dummy_sampled_df()
+    output_d = _get_source_destination_range(df)
+
+    expected_output = {
+        (0.0, 0.0): {"sources_range": 0, "destinations_range": 1},
+        (0.0, 1.0): {"sources_range": 1, "destinations_range": 2},
+        (1.0, 0.0): {"sources_range": 0, "destinations_range": 1},
+        (1.0, 1.0): {"sources_range": 1, "destinations_range": 2},
+        (2.0, 0.0): {"sources_range": 0, "destinations_range": 2},
+        (2.0, 1.0): {"sources_range": 2, "destinations_range": 2},
+    }
+
+    assert output_d == expected_output
+
+
+def test__create_homogeneous_cugraph_dgl_nn_sparse_graph():
+    tensor_d = {
+        "sources_range": 1,
+        "destinations_range": 2,
+        "sources": torch.as_tensor([0, 0, 1, 1], dtype=torch.int64, device="cuda"),
+        "destinations": torch.as_tensor([0, 0, 1, 2], dtype=torch.int64, device="cuda"),
+    }
+
+    seednodes_range = 10
+    sparse_graph = _create_homogeneous_cugraph_dgl_nn_sparse_graph(
+        tensor_d, seednodes_range
+    )
+    assert sparse_graph.num_sources() == 2
+    assert sparse_graph.num_destinations() == 10
```

Comment:
```suggestion
    assert sparse_graph.num_src_nodes() == 2
    assert sparse_graph.num_dst_nodes() == 10
```

Reply:
Thanks for checking this , resolved by https://github.com/rapidsai/cugraph/pull/3810/commits/f5e84961d9a3e7e0cc01b8d26611b02d4daea1ab